src/modules/ChannelSettings/components/UserPanel/user-panel.scss

```diff
@@ -32,5 +32,10 @@
   .sendbird-channel-settings-accordion__footer {
     padding-left: 14px;
     padding-top: 14px;
+    padding-bottom: 14px;
+    @include themed() {
+      background-color: t(bg-0);
+      border-bottom: 1px solid t(on-bg-4);
+    }
```

Comment:
Not related to RTL support actually but just fixed ;) 
**as-is**
![Screenshot 2024-06-24 at 5 36 51 PM](https://github.com/sendbird/sendbird-uikit-react/assets/10060731/54db07cb-db95-49c5-9156-faae86532377)

**to-be**
<img width="317" alt="Screenshot 2024-06-24 at 5 34 53 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/10060731/3e8a66c2-fa5f-472f-8f75-471cd231df90">
<img width="321" alt="Screenshot 2024-06-24 at 5 34 42 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/10060731/75fce434-f205-40d6-8d93-b3ff4510c09e">